demo/dynamic_embedding/movielens-100k-sync-estimator/README.md

```diff
@@ -0,0 +1,16 @@
+# A distributed training demo for `tfra.dynamic_embedding`:
+
+- dataset: [movielen/100k-ratings](https://www.tensorflow.org/datasets/catalog/movielens#movielens100k-ratings)
+- model: DNN
+- Running API: using estimator APIs
+
```

Comment:
Please add version requirements for Horvod and MPI.

Reply:
done.